src/app/home/todos/todos.component.html

```diff
@@ -1,17 +1,19 @@
 <section class="todoapp">
   <header class="header">
     <h1>todos</h1>
-    <input (keydown.enter)="addTodo(input)" #input class="new-todo" placeholder="What needs to be done?" autofocus>
+    <input (keydown.enter)="addTodo(input)" #input class="new-todo" placeholder="{{name}}"
```

Comment:
No need for binding here, this is static text.

Reply:
Yes, static text, I was changing the messages in the input field. I did it because there was no analysis.